packages/editor/src/components/StructureTree/ComponentNode.tsx

```diff
@@ -92,6 +92,14 @@ const ComponentNodeImpl = (props: Props) => {
     e.stopPropagation();
     setIsShowRelationshipModal(true);
   }, []);
+  const onClickShowState = useCallback(
+    (e: React.MouseEvent) => {
+      e.stopPropagation();
+      editorStore.setExplorerMenuTab(3);
```

Comment:
It might be better to use `ExplorerMenuTabs.State`

Reply:
ok